### ISSUE TITLE
real data test

### ISSUE THREAD

**Issue by WillingDil** [caiyuanhao1998/MST#5]
Is the test command wrong?

**Comment by caiyuanhao1998** [comment]
Hi, thanks for your attention

We have not provided pre-trained weights for real exp yet. You can use our code to train real models first and then test.

Until now, we provided the results on real data. Feel free to use our results to make a comparison with yours.

**Comment by luluchou** [comment]
作者是个傻逼！专门盗我号给他自己刷星星！干！

**Comment by Vzhouhan** [comment]
请问您论文里真实实验所用图像的RGB图从哪里获得？

**Comment by caiyuanhao1998** [comment]
数据集下载得到的

**Comment by Vzhouhan** [comment]
在您给出的链接里吗

> 数据集下载得到的

**Comment by bryanbocao** [comment]
Same question here.
The scripts in 5.2 Testing are actually for simulations.

**Comment by dongruoyu1** [comment]
> Same question here. The scripts in 5.2 Testing are actually for simulations.

Do you mean that the script for test in 5.2 is incorrect？

**Comment by bryanbocao** [comment]
@dongruoyu1 I believe so ("test in 5.2 is incorrect"). At least the script didn't work.
5.2 Real Test script looks like a direct copy of the simulation one but doesn't work on real data.

e.g.
```
python test.py --template mst_s --outf ./exp/mst_s/ --method mst_s --pretrained_model_path ./model_zoo/mst/mst_s.pth
```

But the **format** of real measurements is different from simulation so the real test code could not load it.
When I ran the aforementioned code, I got this error:
https://github.com/caiyuanhao1998/MST/issues/36

**Comment by dongruoyu1** [comment]
> @dongruoyu1 I believe so ("test in 5.2 is incorrect"). At least the script didn't work. 5.2 Real Test script looks like a direct copy of simulation, but didn't work on real data.
> 
> e.g.
> 
> ```
> python test.py --template mst_s --outf ./exp/mst_s/ --method mst_s --pretrained_model_path ./model_zoo/mst/mst_s.pth
> ```
> 
> The format of real measurements is different from simulation so the real test code could not load it. When I ran the aforementioned code, I got this error: #36

So what do you think the correct code should be? Have you successfully debugged now?

**Comment by bryanbocao** [comment]
@dongruoyu1 I haven't gone to the real data yet. But the logic is basically to (1) load the real measurements correctly, check the data file's dictionary and the input shape to the model; (2) use a non-reference metric to evaluate the model's prediction as no ground truth is available.

**Comment by Johnweii25** [comment]
> @dongruoyu1 I believe so ("test in 5.2 is incorrect"). At least the script didn't work. 5.2 Real Test script looks like a direct copy of simulation, but didn't work on real data.
> 
> e.g.
> 
> ```
> python test.py --template mst_s --outf ./exp/mst_s/ --method mst_s --pretrained_model_path ./model_zoo/mst/mst_s.pth
> ```
> 
> The format of real measurements is different from simulation so the real test code could not load it. When I ran the aforementioned code, I got this error: #36

also think so
